internal/encoding/crypt3.go

```diff
@@ -20,3 +22,11 @@ func EncodeCrypt3(raw []byte) []byte {
 	dest = append(dest, crypt3Encoding[v&63])
 	return dest
 }
+
+func DecodeInt6(b byte) int {
+	return strings.IndexByte(crypt3Encoding, b)
```

Comment:
Ooh wauw, when I you think you've seen it all. So a single encoded character represents the position in the encoding alphabet, which then represents an int for the cost parameter? I'm just amazed how crypto folks sometimes think that complexity is always better....